docs/operatorTestPlaybooksCI.md

```diff
@@ -0,0 +1,52 @@
+# Purpose
+This document is to describe about the working of the Upstream Operator Test Playbooks CI Pipeline.
+
+# Introduction
+The Operator Test Playbooks CI Pipeline exists to automatically run tests against pull requests in the redhat-operator-ecosystem/operator-test-playbooks GitHub repository.
+
+# Configuration
+A webhook is configured so that the repository sends events to the github2fedmsg web application.
+Any activity (ex: adding comments) in the repository is therefore sent to github2fedmsg, which then rebroadcasts on the Fedora Messaging bus (fedmsg). 
+
+# Usage
+* The [test-operator-test-playbooks](https://jenkins-cvp-ci.cloud.paas.psi.redhat.com/job/test-operator-test-playbooks/) job runs on the CVP CI Jenkins server.
+
+* The job is triggered by the Fedora Messaging messages which:
```

Comment:
`...by a Fedora Messaging message which` or `by a message on the Fedora Messaging bus`